src/traces/sankey/render.js

```diff
@@ -220,23 +124,55 @@ function linkModel(uniqueKeys, d, l) {
     };
 }
 
-function nodeModel(uniqueKeys, d, n) {
+function linkPath() {
+    var curvature = 0.5;
+    function path(d) {
+        if(d.circular) {
+            return d.link.path; // TODO: turn this into a closed path to support link.line.(width|color)
+        } else {
+            var x0 = d.link.source.x1;
```

Comment:
Why do we need this new `linkPath` logic? I can't find this anymore in the `master` version of this file.

Reply:
Basically, what's in the `else` clause used to live in our fork of `d3-sankey`.

Reply:
I added even more code to generate SVG path in d1d852ab3a4270d3b7a847ba84b6b0ae5e43e967 just above that function.

The reason it's in here is because `d3-sankey` and `d3-sankey-circular` use open paths with a large `stroke-width` to render links whereas we use closed path that are filled. There are pros and cons to both approaches. You can see this PR that was refused for more details https://github.com/d3/d3-sankey/pull/24 . Therefore, because we have our own take on how to render those links, that logic should probably live in our repo.